classes/Members/class.ilUserTakeOverMembersGUI.php

```diff
@@ -175,7 +175,10 @@ protected function fillForm(&$form)
     protected function searchUsers()
     {
         // Only Administrators
-        if (!in_array(2, self::dic()->rbacreview()->assignedGlobalRoles(self::dic()->user()->getId()))) {
+
+        if (!usrtoHelper::getInstance()->checkPluginAccess())
+       // if (!in_array(2, self::dic()->rbacreview()->assignedGlobalRoles(self::dic()->user()->getId())))
```

Comment:
What ist this, why is it outcommented, -> probably remove.

Reply:
fixed